src/client/src/apps/MainRoute/widgets/blotter/components/blotterUtils.ts

```diff
@@ -19,7 +19,7 @@ import {
 } from '../blotterFields'
 
 function UtcFormatDate(date: Date) {
-  const localZoneName = DateTime.local().zoneName
+  const localZoneName = Info.features().zones ? DateTime.local().zoneName : 'utc'
```

Comment:
Can we log to the console what time zone is detected and used? May be useful for debugging in the future.